contracts/broker-staking/src/contract.rs

```diff
@@ -59,9 +59,38 @@ pub fn execute(
         ExecuteMsg::WithdrawAll { to } => {
             withdraw_all(deps, env, info, to, contract_addr)
         }
+        ExecuteMsg::ClaimRewards {} => claim_rewards(deps, env, info),
     }
 }
 
+pub fn claim_rewards(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+) -> Result<Response, ContractError> {
+    let is_halted = IS_HALTED.load(deps.storage)?;
+    assert_not_halted(is_halted)?;
+
+    Permissions::assert_operator(deps.storage, info.sender.to_string())?;
+
+    // query current delegations
+    let delegations =
+        deps.querier.query_all_delegations(&env.contract.address)?;
+
+    let mut messages: Vec<CosmosMsg> = vec![];
+    for delegation in delegations.iter() {
```

Comment:
There's a CLI command to do withdraw all but it's internally just doing this logic. There's no actual message to withdraw from all delegations in the comsos sdk